autogen/main/cluster.tf.tmpl

```diff
@@ -530,6 +530,21 @@ resource "google_container_cluster" "primary" {
         }
       }
 
+      dynamic "kubelet_config" {
+        for_each = length(setintersection(
+          keys(each.value),
+          ["cpu_manager_policy", "cpu_cfs_quota", "cpu_cfs_quota_period", "insecure_kubelet_readonly_port_enabled", "pod_pids_limit"]
+        )) != 0 ? [1] : []
+
+        content {
+          cpu_manager_policy                     = lookup(each.value, "cpu_manager_policy", "static")
```

Comment:
the lint warning caught it, but iterator has to be `kubelet_config` (or set to something else explicitly) here vs `each`

Reply:
Yeah, this actually needs to be off `var.node_pools[0]`.